src/main/java/net/sf/jabref/importer/fileformat/MedlineImporter.java

```diff
@@ -37,7 +38,7 @@
  * Importer for the Medline format.
  *
  * check here for details on the format
- * --INSERT INFO HERE--
+ * http://www.nlm.nih.gov/bsd/mms/medlineelements.html
```

Comment:
that's the wrong link, you need the link for the [xml elements] (https://www.nlm.nih.gov/bsd/licensee/elements_descriptions.html).